tests/test_api_security.py

```
@@ -57,7 +57,9 @@ def test_cache_key_stability(self):
         mock_response.json.return_value = {"data": "success"}
         mock_response.encoding = "utf-8"
         # Use side_effect to return a fresh iterator for each call
-        mock_response.iter_content.side_effect = lambda chunk_size=None: iter([b'{"data": "success"}'])
+        mock_response.iter_content.side_effect = lambda chunk_size=None: iter(
```

Ruff (0.14.14)

[warning] Unused lambda argument: `chunk_size`

(ARG005)